src/region.js

```diff
@@ -269,7 +275,11 @@ const Region = MarionetteObject.extend({
     delete this.currentView;
 
     if (!view._isDestroyed) {
-      this._removeView(view, shouldDestroy);
+      if (shouldDestroy) {
+        this.removeView(view);
+      } else {
+        this._detachView(view);
```

Comment:
I suppose what I mean is that calling `myRegion.empty()` might animate out the view, but calling `myRegion.detachView()` would not.